phlay

```diff
@@ -503,13 +503,13 @@ class Revision(Cached):
             '', self.info(conduit)['fields']['summary'], count=1
         ).strip()
 
-    def edges(self, conduit):
+    def parents(self, conduit):
```

Comment:
Good catch. This used to be a much more generic edge searching function, but this makes more sense for its actual purpose.